drivers/charger/charger_max20335.c

```diff
@@ -93,6 +116,37 @@ static int max20335_get_status(const struct device *dev, enum charger_status *st
 	return 0;
 }
 
+static int max20335_get_charger_online(const struct device *dev, enum charger_online *online)
+{
+	enum {
+		MAX20335_CHGIN_IN_NOT_PRESENT_OR_INVALID,
+		MAX20335_CHGIN_IN_PRESENT_AND_INVALID,
+	};
+	const struct charger_max20335_config *const config = dev->config;
+	uint8_t val;
+	int ret;
+
+	ret = i2c_reg_read_byte_dt(&config->bus, MAX20335_REG_STATUS_B, &val);
+	if (ret) {
+		return ret;
+	}
+
+	val = FIELD_GET(MAX20335_STATUS_B_USB_OK_MASK, val);
+
+	switch (val) {
+	case MAX20335_CHGIN_IN_NOT_PRESENT_OR_INVALID:
```

Comment:
You can omit this value and immediately go to the default value:

<pre>
	switch (val) {
	case MAX20335_CHGIN_IN_PRESENT_AND_INVALID:
		*online = CHARGER_ONLINE_FIXED;
		break;
	default:
		*online = CHARGER_ONLINE_OFFLINE;
		break;
	};
</pre>